Makefile

```diff
@@ -7,7 +7,7 @@ BUILD_DATE := $(shell date -u +%FT%TZ)
 ETLHASH=stellar/stellar-etl:$(shell git rev-parse --short HEAD)
 
 docker-build:
-	$(SUDO) docker build --pull --no-cache --label org.opencontainers.image.created="$(BUILD_DATE)" \
+	$(SUDO) docker build --platform linux/amd64 --pull --no-cache --label org.opencontainers.image.created="$(BUILD_DATE)" \
```

Comment:
I recommend not adding `--platform` to the Makefile as it is used in our https://github.com/stellar/pipelines/blob/master/stellar-etl/Jenkinsfile#L44-L45